### PR TITLE
update rootlesskit to 1.0.1

### PR DESCRIPTION
**Pull request by dirkmueller**
This avoids an issue with u-root 7.0.0 which has been pulled:

  $ go list -u -m all
  $ go list -m: github.com/u-root/u-root@v7.0.0+incompatible: retracted by module author: Published v7 too early (before migrating to go modules)

#### Proposed Changes ####

This enables outdated-dependency check to work again and not fail on a retracted module.

#### Types of Changes ####

Bugfix

#### Verification ####

CI passes

#### Linked Issues ####

* TBD

#### User-Facing Change ####

```release-note
Bumped rootlesskit to v1.0.1
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -104,7 +104,7 @@ require (
 	github.com/rancher/wharfie v0.5.1
 	github.com/rancher/wrangler v1.0.0
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/rootless-containers/rootlesskit v0.14.5
+	github.com/rootless-containers/rootlesskit v0.14.6
```
Comment (AkihiroSuda) [inline go.mod:107]: Any reason not to use v1.0.1?

**Comment by dirkmueller** [comment]
so it looks like 1.0.1 is failing on the arm architecture. going back to 0.14.6 to confirm

**Comment by AkihiroSuda** [comment]
> so it looks like 1.0.1 is failing on the arm architecture. going back to 0.14.6 to confirm

What was the error?

**Comment by brandond** [comment]
I think the failure with 1.0.1 was just a CI flake. Your other PR for 1.x.x seems to be passing at the moment.

**Comment by brandond** [comment]
I'm a little confused, you appear to be updating both this PR and https://github.com/k3s-io/k3s/pull/5788 - which one would you like to move forward with?

**Comment by dirkmueller** [comment]
> I'm a little confused, you appear to be updating both this PR and #5788 - which one would you like to move forward with?

the other one was just for testing the CI. You were right, the previous failure appeared to be a CI fluke, so this is good to go.

**Review by brandond — APPROVED** [review]

**Pull request merged**